lib/amazon-pricing/definitions/instance-type.rb

```diff
@@ -610,7 +612,7 @@ def self.get_values(json, category_type, override_price = false)
       'i3.4xlarge'  => [ 437, 16000], # EBSOptimized
       'i3.8xlarge'  => [ 875, 32500], # EBSOptimized
       'i3.16xlarge' => [1750, 65000], # EBSOptimized
-      'i3.metal'    => [1250, 64000], # EBSOptimized
+      'i3.metal'    => [1750, 65000], # EBSOptimized
       'i3p.16xlarge' => [1250, 64000], # EBSOptimized
```

Comment:
shouldn't `i3p.16xlarge` match this `i3.metal` change?

Reply:
good question; I didn't see it on their published throughput page, but I remember now we had these two having the same values as i3p.16xl is essentially similar to the i3.metal. I'll update.